File: blingbling/src/main/java/dinson/customview/weight/recycleview/CommonViewHolder.java
package dinson.customview.weight.recycleview;

import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

/**
 * 单一布局通用viewholder
 */
public class CommonViewHolder extends RecyclerView.ViewHolder {
    public CommonViewHolder(View itemView) {
        super(itemView);
    }

    /**
     * 加载layoutId视图并用LGViewHolder保存
     */
    static CommonViewHolder getViewHolder(ViewGroup parent, int layoutId) {
        View itemView = LayoutInflater.from(parent.getContext()).inflate(layoutId, parent, false);
        return new CommonViewHolder(itemView);
    }
}

File: blingbase/src/main/java/com/dinson/blingbase/widget/recycleview/CommonViewHolder.kt
package com.dinson.blingbase.widget.recycleview

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView

/**
 * 单一布局通用viewholder
 */
class CommonViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    companion object {
        /**
         * 加载layoutId视图并用LGViewHolder保存
         */
        fun getViewHolder(parent: ViewGroup, layoutId: Int): CommonViewHolder {
            val itemView = LayoutInflater.from(parent.context).inflate(layoutId, parent, false)
            return CommonViewHolder(itemView)
        }
    }
}